test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInListenerTests.cs

```diff
@@ -132,11 +133,16 @@ public void Dispose()
                 new ActivityContext(
                     expectedTraceId,
                     expectedSpanId,
-                    ActivityTraceFlags.Recorded),
+                    ActivityTraceFlags.Recorded,
+                    isRemote: true),
                 default));
 
-            var activity = new Activity(ActivityNameAspNet).AddBaggage("Stuff", "123");
-            activity.SetParentId(expectedTraceId, expectedSpanId, ActivityTraceFlags.Recorded);
+            var activity = new Activity(HttpInListener.ActivityOperationName).AddBaggage("Stuff", "123");
```

Comment:
wondering what is the relevance of this AddBaggage? (its not newly added in this pr anyway)

Reply:
🤷 There were those few days where we had the Baggage API built on top of Activity.Baggage, maybe this is a carry-over from then? I don't think it has any relevance to these tests at the moment.

Reply:
Got it. Will cleanup as part of overall cleanups I plan to do in next few days!